### PR TITLE
fix bundle page

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
ADO card: ☑️ [AB#9825](https://dev.azure.com/S72/fefacba9-96b6-4af2-a53d-050ed453e13a/_workitems/edit/9825)

Noticed yesterday a bundle page style regression, just want to add a fix

Bundle page should look like:
https://staging-event.shift72.com/bundle/the-matrix-trilogy/

Currently looks like:
https://staging-store-kibble.shift72.com/bundle/super-bundle/

### CODE REVIEW

**Review by additiverich — APPROVED** [review]
Is this going to be another battlefield in the grid/flexbox holy war

File: CHANGELOG.md
Comment (PeculiarGoat) [inline CHANGELOG.md:23]: I'm not sure what it is exactly... but something doesn't feel right about putting ADO card numbers in a public changelog.

File: site/styles/_meta-detail.scss
Comment (PeculiarGoat) [inline site/styles/_meta-detail.scss:192]: Run the linter please!

Comment (PeculiarGoat) [reply site/styles/_meta-detail.scss:192]: Also, I think `grid-column: 2;` is clearer than `grid-column-end: -1;` since we know there are two columns by design.  Cool trick though... I didn't know about negative grid values!

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
Tested and it works all good... just a couple of arguments to have before approval!  :P

File: CHANGELOG.md
Comment (matthewthomsonnz) [reply CHANGELOG.md:23]: @l0ud0gg what do you reckon?

File: site/styles/_meta-detail.scss
Comment (matthewthomsonnz) [reply site/styles/_meta-detail.scss:191]: It's on grid-column 1 under the lg breakpoint (you'll see the layout breaks).

It's pretty neat and saves you specifying grid-column-start as it grows to take up the entire space. Essentially flex-grow 1.

Comment (PeculiarGoat) [reply site/styles/_meta-detail.scss:191]: Hold up I've confused myself thinking this CSS is shared by collections on the film detail page (where I would have been wrong about the column also lol).  Looks like collections on the film detail page live outside of `meta-detail-main`... would it be better to change the bundle page structure to match the film page?

Comment (matthewthomsonnz) [reply site/styles/_meta-detail.scss:191]: I tried it out and it seems like a good idea for consistency.

Only thing it needs the padding from swiper-container class so I'll wait for the collection-container PR merge

Comment (matthewthomsonnz) [reply site/styles/_meta-detail.scss]: @PeculiarGoat padding added, ready for test and approval

**Review by PeculiarGoat — APPROVED** [review]
Looks good nice work!

**Pull request merged**
